.github/workflows/docker.yaml

```
@@ -0,0 +1,49 @@
+name: Build and public Docker image
+
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+    branches:
+      - 'main'
```

```suggestion
  pull_request: {}
```

I thought to run this for all PRs it would need to be this, am I wrong?

---

I think you're right. What I wrote was meant to be "generate it for the pull requests that target a merge to the `main` branch". But I don't have any preference, we can do it the same way you did 🙂

---

Updated.